samples/compose-samples/src/main/java/com/squareup/sample/compose/launcher/SampleLauncherApp.kt

```diff
@@ -148,14 +148,15 @@ private fun launchSample(
   startActivity(context, intent, options)
 }
 
-private fun Modifier.disableTouchInput(): Modifier = pointerInput(Unit) {
-  forEachGesture {
-    awaitPointerEventScope {
-      awaitPointerEvent(Initial).let { event ->
-        event.changes.forEach { change ->
-          if (change.pressed != change.previousPressed) change.consume()
+private fun Modifier.disableTouchInput(): Modifier =
+  then(
```

Comment:
You don't need `then` when calling other modifier factories that already properly chain. This actually ends up calling `then` twice which i'm surprised doesn't have negtive side effects.